xml/System.Net.Http/HttpClient.xml

```diff
@@ -1431,11 +1443,23 @@ The `DefaultRequestVersion` property can be changed as long as the <xref:System.
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="requestUri">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="requestUri">The Uri the request is sent to.</param>
+        <param name="cancellationToken">The cancellation token to cancel the operation.</param>
+        <summary>Send a GET request to the specified Uri and return the response body as a byte array in an asynchronous operation.</summary>
+        <returns>The task object representing the asynchronous operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response body is read.  
+
+> [!NOTE]
+> In case of timeout, different exceptions are thrown on different .NET implementations.
+> - <xref:System.Net.Http.HttpRequestException> is thrown on all applicable .NET Framework versions.
+> - <xref:System.Threading.Tasks.TaskCanceledException> without any inner exception is thrown on all applicable .NET Core versions.
+> - <xref:System.Threading.Tasks.TaskCanceledException> that nests a <xref:System.TimeoutException> is thrown on .NET 5.0 and later versions.
+ ]]></format>
+        </remarks>
```

Comment:
```suggestion

> [!NOTE]
> In case of a timeout:
> - .NET Core throws a <xref:System.Threading.Tasks.TaskCanceledException> without any inner exception.
> - .NET 5.0 and later versions throw a <xref:System.Threading.Tasks.TaskCanceledException> that nests a <xref:System.TimeoutException>.

          ]]></format>
        </remarks>
        <exception cref="T:System.Net.Http.HttpRequestException">.NET Framework only: The operation timed out.</exception>
        <exception cref="T:System.Threading.Tasks.TaskCanceledException">.NET Core and .NET 5.0+: The operation timed out.</exception>
```